lib/matplotlib/axes/_axes.py

```diff
@@ -8298,8 +8298,9 @@ def matshow(self, Z, **kwargs):
         return im
 
     @_preprocess_data(replace_names=["dataset"])
-    def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
-                   showmeans=False, showextrema=True, showmedians=False,
+    def violinplot(self, dataset, positions=None, vert=None,
+                   orientation='vertical', widths=0.5, showmeans=False,
+                   showextrema=True, showmedians=False,
                    quantiles=None, points=100, bw_method=None, side='both'):
```

Comment:
`orientation` can't be inserted in the middle here to prevent breaking passing all arguments positionally. I believe that will be fixed by #27786.

Reply:
Since https://github.com/matplotlib/matplotlib/pull/27786 is now merged, should we put `orientation` in the place where `vert` was?